src/transformStateWithClones.js

```diff
@@ -65,7 +65,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
+  const cloneState = { ...state };
+
+  for (const transform of transforms) {
+    const properties = transform.properties;
```

Comment:
Використай деструктуризацію
```suggestion
    const { properties } = transform;
```